packages/genesys-spark-tokens/snapshot-new/json/gse-legacy-light-ui.json

```diff
@@ -1345,7 +1345,7 @@
     "box_shadow": {
       "blur": "4px",
       "spread": "1px",
-      "color": "rgba(35,57,92,0.1)",
+      "color": "#23395c",
```

Comment:
These colours look different to me, is there a reason those are changing as part of this?

Reply:
Fixed